src/App.jsx

```diff
@@ -0,0 +1,53 @@
+/* eslint linebreak-style: ["error", "windows"] */
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import Navbar from './components/navbar/Navbar';
+import AboutScreen from './routes/About';
+import AdminScreen from './routes/admin/Admin';
+import BookingScreen from './routes/Booking';
+import DetailScreen from './routes/Detail';
+import EditScreen from './routes/admin/Edit';
+import HomeScreen from './routes/Home';
+import NewScreen from './routes/admin/New';
+import ReservationScreen from './routes/Reservation';
+import LoginScreen from './routes/Login';
+import SignupScreen from './routes/Signup';
+import ProtectedRoutes from './routes/ProtectedRoutes';
+
+function App() {

```

Comment:
- [ ] There are multiple linter errors throughout your app.  If you fix them, chances are your app will run smoothly.